src/utils/websocket.js

```
@@ -192,6 +192,71 @@ export function getStatusIcon(resource, type = '') {
+
+  switch (status) {
+    case 'connecting':
+      src = wsConnecting
+      break
+    case 'open':
+      src = wsOpen
+      break
+    case 'closed':
+      src = wsClosed
+      break
+    case 'error':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)